developer-tools.md

```diff
@@ -397,6 +397,18 @@ Other tips:
 - "Rebuild Project" can fail the first time the project is compiled, because generate source files 
 are not automatically generated. Try clicking the "Generate Sources and Update Folders For All 
 Projects" button in the "Maven Projects" tool window to manually generate these sources.
+- Maven bundled in IntelliJ may not meet the minimum version requirement of the Spark. If that happens,
```

Comment:
"The version of Maven bundled with IntelliJ may not be new enough for Spark. ..."

Reply:
Thanks for the review. Yes. I am going to change it.